dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -7983,6826 +7983,9449 @@ BEGIN
-    42799 "Latin Small Letter Cuatrillo With Comma"
-    42800 "Latin Letter Small Capital F"
-    42801 "Latin Letter Small Capital S"
-    42802 "Latin Capital Letter Aa"
-    42803 "Latin Small Letter Aa"
-    42804 "Latin Capital Letter Ao"
-    42805 "Latin Small Letter Ao"
-    42806 "Latin Capital Letter Au"
-    42807 "Latin Small Letter Au"
-    42808 "Latin Capital Letter Av"
-    42809 "Latin Small Letter Av"
-    42810 "Latin Capital Letter Av With Horizontal Bar"
-    42811 "Latin Small Letter Av With Horizontal Bar"
-    42812 "Latin Capital Letter Ay"
-    42813 "Latin Small Letter Ay"
-    42814 "Latin Capital Letter Reversed C With Dot"
-    42815 "Latin Small Letter Reversed C With Dot"
-    42816 "Latin Capital Letter K With Stroke"
-    42817 "Latin Small Letter K With Stroke"
-    42818 "Latin Capital Letter K With Diagonal Stroke"
-    42819 "Latin Small Letter K With Diagonal Stroke"
-    42820 "Latin Capital Letter K With Stroke And Diagonal Stroke"
-    42821 "Latin Small Letter K With Stroke And Diagonal Stroke"
-    42822 "Latin Capital Letter Broken L"
-    42823 "Latin Small Letter Broken L"
-    42824 "Latin Capital Letter L With High Stroke"
-    42825 "Latin Small Letter L With High Stroke"
-    42826 "Latin Capital Letter O With Long Stroke Overlay"
-    42827 "Latin Small Letter O With Long Stroke Overlay"
-    42828 "Latin Capital Letter O With Loop"
-    42829 "Latin Small Letter O With Loop"
-    42830 "Latin Capital Letter Oo"
-    42831 "Latin Small Letter Oo"
-    42832 "Latin Capital Letter P With Stroke Through Descender"
-    42833 "Latin Small Letter P With Stroke Through Descender"
-    42834 "Latin Capital Letter P With Flourish"
-    42835 "Latin Small Letter P With Flourish"
-    42836 "Latin Capital Letter P With Squirrel Tail"
-    42837 "Latin Small Letter P With Squirrel Tail"
-    42838 "Latin Capital Letter Q With Stroke Through Descender"
-    42839 "Latin Small Letter Q With Stroke Through Descender"
-    42840 "Latin Capital Letter Q With Diagonal Stroke"
-    42841 "Latin Small Letter Q With Diagonal Stroke"
-    42842 "Latin Capital Letter R Rotunda"
-    42843 "Latin Small Letter R Rotunda"
-    42844 "Latin Capital Letter Rum Rotunda"
-    42845 "Latin Small Letter Rum Rotunda"
-    42846 "Latin Capital Letter V With Diagonal Stroke"
-    42847 "Latin Small Letter V With Diagonal Stroke"
-    42848 "Latin Capital Letter Vy"
-    42849 "Latin Small Letter Vy"
-    42850 "Latin Capital Letter Visigothic Z"
-    42851 "Latin Small Letter Visigothic Z"
-    42852 "Latin Capital Letter Thorn With Stroke"
-    42853 "Latin Small Letter Thorn With Stroke"
-    42854 "Latin Capital Letter Thorn With Stroke Through Descender"
-    42855 "Latin Small Letter Thorn With Stroke Through Descender"
-    42856 "Latin Capital Letter Vend"
-    42857 "Latin Small Letter Vend"
-    42858 "Latin Capital Letter Et"
-    42859 "Latin Small Letter Et"
-    42860 "Latin Capital Letter Is"
-    42861 
```

Comment:
I am all for translating as much as possible, but I am not sure „Tiny” and „Miny” (as mathematical operators used in combinatorial game theory) should. See [Tiny and miny on Wikipedia](https://en.wikipedia.org/wiki/Tiny_and_miny)